#49 - Ignore Invalid SSL Cert

zbb93 opened the pull request:
Added new command line flag '--unsafe'. If this flag is used a tlsConfig is created for the transporter with InsecureSkipVerify set to true.

georgijd commented:
Since we borrow command line parameter names from cURL, I think this parameter should be called `--insecure` instead of `--unsafe` :)

zbb93 commented:
Ah, I didn't realize you guys were following cURL. I have changed the name of the parameter to `--insecure` now. However, it appears the build is failing? The only output I see is "The command "test -z "$(gofmt -l ./)"" exited with 1". I can't find any tests in the repo though. Does anyone have any insight into what is causing the build to fail?

Benaiah commented:
@zbb93 it sounds like you didn't run `gofmt` on your code before you committed, so Travis is flagging it for code style. You should be able to just `gofmt` it and be good.

asciimoo commented:
Yap, it is formatting error.
@zbb93 could you rebase the code to the latest master and fix the formatting issue?

zbb93 commented:
Thank you guys for your help! I have fixed the formatting issue and I added support for an `insecure` parameter in the config file. Let me know if there are any further changes you would like made for this.

asciimoo commented:
@zbb93 seems, something went wrong with the rebase. The diff contains previous changes.

zbb93 commented:
I'm not sure if I accidentally closed it or if me trying to reset my branch caused this to get closed. I'm still working on the changes though. Sorry for the delay, fighting with git.
